project_name/urls.py

```diff
@@ -22,12 +22,15 @@
 from django.views.generic import TemplateView
 
 from geonode.urls import urlpatterns
+from .views import (get_categorys, get_most_maps,)
 
 urlpatterns += (
 ## include your urls here
     url(r'^aboutus/$', 
   	    TemplateView.as_view(template_name='aboutus.html'),
         name='aboutus'),
+    url(r'^getCategory/', get_categorys, name='getCategory'),
```

Comment:
please note that GeoNode has already an API, which can be used in this case. For example, for Categories: http://yourgeonode/api/categories/